scripts/adaptBuildToFf.mts

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs/promises';
+
+const BUILT_MANIFEST_PATH = './packages/extension/build/manifest.json';
+
+fs.readFile(BUILT_MANIFEST_PATH, 'utf8').then(chromeContent => {
+  const { background: { service_worker, ...background }, ...json } = JSON.parse(chromeContent)
+
+  const ffJson = {
+    ...json,
+    background: {
+      scripts: [service_worker],
+      ...background,
```

Comment:
so cool! running this script multiple times doesn't change the results :heart:

Reply:
heh, that's an accident :P but I can leave it like that :D